Documentation/Security/GuidelinesIntegrators/InstallTool.rst

```diff
@@ -30,6 +30,12 @@ need write access to this directory on a server level (for example via SSH, SFTP
 or you can create this file as a backend user with administrator
 privileges.
 
+..  attention::
+    Even if you run TYPO3 v12 in the Composer-mode at the time of writing
```

Comment:
```suggestion
    Even if you run TYPO3 v12 in Composer mode at the time of writing
```

Had to read it multiple times, it sounds strange without some punctuation, maybe better:

Even if you run TYPO3 v12 in the Composer-mode, the file has still to be created in folder public/typo3conf/ENABLE_INSTALL_TOOL (as at the time of writing TYPO3 v12.0.0).

But I am not sure if we have to mention it. Do you create the file before accessing the install tool without being logged-in into the backend? I always call install.php, copy the path (as I mostly don't remember the name) and make a "touch" on file system. I think, the most people will do it that way. And I assume this file will be moved into the public/project folder, so this note becomes obsolete later. Maybe we should just add a todo with a link to the forge ticket to pick it up later.